ompi/runtime/ompi_mpi_params.c

```diff
@@ -263,12 +263,11 @@ int ompi_mpi_register_params(void)
         ompi_rte_abort(1, NULL);
     }
 
-    ompi_add_procs_cutoff = 1024;
```

Comment:
I think we should set the variable back to the default. That is the way I have done it pretty much everywhere throughout the code base. See the other registrations in this same function.

If we come through registration twice it means we finalized the project or component. I don't think we should use a possibly stale value.